README.md

```diff
@@ -188,17 +188,29 @@ git config blame.ignoreRevsFile .git-blame-ignore-revs
 
 On certain pages an API endpoint is called to receive experiment analysis data from Jetstream to display visualization tables. To see experiment visualization data, you must provide GCP credentials.
 
-1. Generate a GCP private key file.
+0. *prerequisite* Install GCP CLI
+    - Follow the instructions [here](https://cloud.google.com/sdk/docs/install)
+    - Project: `moz-fx-data-experiments`
+
+1. Authorize CLI with your account
+    - `gcloud auth login`
```

Comment:
as discussed the account needs to have `storage.objects.list` permissions on `moz-fx-data-experiments`